templates/new/batch_main.py.txt

```diff
@@ -1,22 +1,54 @@
 import argparse
-from surround import Assembler
-from stages import Main, ValidateData
+from surround import Assembler, RunMode
+from stages import Main, InputValidator
 from batch_runner import BatchRunner
 
+assemblies = [
+    Assembler("Default")
```

Comment:
Can you please rename `Default` to `baseline`? The first element in the array can always be the baseline assembly for every project and lowercase to make it easier to type from the command line.

Reply:
Sure :) also the `run` command is insensitive to case, so `Baseline` will match with `baseline` when running the command

Reply:
Done